.changeset/old-otters-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Added: `--ext` and `extensions` to the Node.js API
```

Comment:
[suggest] I've updated the PR title. Please follow it here:


```suggestion
Added: `--ext` CLI flag and `extensions` Node.js API option
```

Perhaps `extensions` may be renamed to `fileExtensions` though.